proposals/sar_backscatter.json

```diff
@@ -0,0 +1,157 @@
+{
+    "id": "sar_backscatter",
+    "summary": "Computes backscatter from SAR input",
+    "description": "Computes backscatter from SAR input.\n\nNote that backscatter computation may require instrument specific metadata that is tightly coupled to the original SAR products. As a result, this process may only work in combination with loading data from specific collections, not with general data cubes.",
+    "categories": [
+        "cubes",
+        "sar"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The source data cube containing SAR input.",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "name": "coefficient",
+            "description": "The backscatter coefficient to compute, either `sigma0` or `gamma0` (the default).",
+            "optional": true,
+            "default": "gamma0",
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "sigma0",
+                    "gamma0"
+                ]
+            }
+        },
+        {
+            "name": "orthorectify",
+            "description": "By default, orthorectification is disabled. Set to `true` to enable orthorectification.\n\nThe non-orthorectified products use a simple earth model as provided in the products themselves. This may be sufficient for very flat target areas and is faster to process.",
```

Comment:
Discription says "By default, orthorectification is disabled" but default value in spec is `true`

Reply:
Thank you, fixed.